--- pacparser-1.2.5/src/Makefile~	2010-09-13 08:44:24.000000000 +0200
+++ pacparser-1.2.5/src/Makefile	2010-09-14 08:59:29.728966797 +0200
@@ -74,25 +74,18 @@
 spidermonkey/js/src: spidermonkey/js-?.?.?.tar.gz
 	tar xzvf spidermonkey/js-?.?.?.tar.gz -C spidermonkey
 
-jsapi_buildstamp: spidermonkey/js/src
-	cd spidermonkey && SMCFLAGS="$(SHFLAGS) $(SMCFLAGS)" $(MAKE) jsapi
-	touch jsapi_buildstamp
-
-libjs.a: spidermonkey/js/src
-	cd spidermonkey && SMCFLAGS="$(SHFLAGS) $(SMCFLAGS)" $(MAKE) jslib
-
-pacparser.o: pacparser.c pac_utils.h pacparser.h jsapi_buildstamp
+pacparser.o: pacparser.c pac_utils.h pacparser.h
 	$(CC) $(CFLAGS) $(SHFLAGS) -c pacparser.c -o pacparser.o
 	touch pymod/pacparser_o_buildstamp
 
-$(LIBRARY): pacparser.o libjs.a
-	$(MKSHLIB) $(LIB_OPTS) -o $(LIBRARY) pacparser.o libjs.a $(LDFLAGS)
+$(LIBRARY): pacparser.o
+	$(MKSHLIB) $(LIB_OPTS) -o $(LIBRARY) pacparser.o $(LDFLAGS)
 
 $(LIBRARY_LINK): $(LIBRARY)
 	ln -sf $(LIBRARY) $(LIBRARY_LINK)
 
 pactester: pactester.c pacparser.h $(LIBRARY_LINK)
-	$(CC) pactester.c -o pactester -lpacparser -L. -I.
+	$(CC) $(CFLAGS) $(LDFLAGS) pactester.c -o pactester -lpacparser -L. -I.
 
 testpactester: pactester
 	echo "Running tests for pactester."
