--- pacparser-1.1.1/Makefile~	2010-06-19 10:17:44.000000000 +0200
+++ pacparser-1.1.1/Makefile	2010-07-30 10:41:23.360530968 +0200
@@ -54,24 +54,18 @@
 .PHONY: clean pymod install-pymod
 all: pactester
 
-jsapi: spidermonkey/js.tar.gz
-	cd spidermonkey && $(MAKE) jsapi
-
-libjs.a: spidermonkey/js.tar.gz
-	cd spidermonkey && $(MAKE) jslib
-
-pacparser.o: pacparser.c pac_utils.h jsapi
+pacparser.o: pacparser.c pac_utils.h
 	$(CC) $(CFLAGS) $(SHFLAGS) -c pacparser.c -o pacparser.o
 	touch pymod/pacparser_o_buildstamp
 
-$(LIBRARY): pacparser.o libjs.a
-	$(MKSHLIB) $(LIB_OPTS) -o $(LIBRARY) pacparser.o libjs.a $(LDFLAGS)
+$(LIBRARY): pacparser.o
+	$(MKSHLIB) $(LIB_OPTS) -o $(LIBRARY) pacparser.o $(LDFLAGS)
 
 libpacparser.$(SO_SUFFIX): $(LIBRARY)
 	ln -sf $(LIBRARY) libpacparser.$(SO_SUFFIX)
 
 pactester: pactester.c pacparser.h libpacparser.$(SO_SUFFIX)
-	$(CC) pactester.c -o pactester -lpacparser -L. -I.
+	$(CC) $(CFLAGS) $(LDFLAGS) pactester.c -o pactester -lpacparser -L. -I.
 
 install: all
 	install -d $(LIB_PREFIX) $(INC_PREFIX) $(BIN_PREFIX)
