--- pacparser-1.0.8/Makefile~	2009-11-25 12:07:37.000000000 +0000
+++ pacparser-1.0.8/Makefile	2009-12-03 13:36:09.584197153 +0000
@@ -60,8 +60,8 @@
 endif
 
 ifndef SM_INC
-ifeq (yes, $(shell [ -e /usr/local/include/js ] && echo yes))
-  SM_INC = -I/usr/local/include/js
+ifeq (yes, $(shell [ -e /usr/include/js ] && echo yes))
+  SM_INC = -I/usr/include/js
 else
   ifeq (yes, $(shell [ -e /opt/local/include/js ] && echo yes))
     SM_INC = -I/opt/local/include/js
