--- pacparser-1.3.0/src/pymod/setup.py.orig	2011-09-10 08:36:51.000000000 +0200
+++ pacparser-1.3.0/src/pymod/setup.py	2011-09-18 17:26:31.571407941 +0200
@@ -42,9 +42,9 @@
     return
 
   pacparser_module = Extension('_pacparser',
-                               include_dirs = ['../spidermonkey/js/src', '..'],
+                               include_dirs = ['..', '/usr/include/js'],
                                sources = ['pacparser_py.c'],
-                               extra_objects = ['../pacparser.o', '../libjs.a'])
+                               libraries = ['pacparser', 'js'])
   setup (name = 'pacparser',
          version = '1',
          description = 'Pacparser package',
